doc/logo.md

```diff
@@ -0,0 +1,177 @@
+
+LISP LOGO is available in well-commented interpretive code. Timestamps for the TVRTLE file
+versions in the five hundreds are from 1978-1979.
+
+
+### MIT 11LOGO
+
+MIT 11LOGO is the LOGO system implemented for the PDP 11/45 at the MIT Artificial
+Intelligence Laboratory. We can run it under PDP-10 / ITS under emulation.
+
+to run 11LOGO type:
+```
+*:pdp45
+1145.427
+CORE = 8.K
+!56.:core
+CORE = 56.K
+!;a pk
+!;a df
+!;l system bin
+!400;g
+11LOGO 1007
+?STARTDISPLAY
+```
+The emulator must be run on the KA10 simulator which supports the Type 340 display.
+
+11LOGO system was one of the major logo versions of the 70s and included not only
+the LOGO evaluator but also a dedicated time-sharing system which serviced about
+dozen users. MIT AI MEMO 313 cites among the system developers Ron Label, Joe Cohen,
+Nat Goodman, Hal Abelson, Roger Hale, Radia Perlman. The 11LOGO display controller
+was designed and built by Tom Knight. The document also cited contributions of Richard
+Greenblatt on matters of system design and, of course, Seymour Papert and Cynthina
+Solomon on language specification.
+
+But please note, that the 11LOGO version 1005 currently available on ITS is very early
+MIT 11LOGO one. It is in essence just logo language evaluator with limited turtle display
+340 grapics capabilities only. A brief description of the built-in procedures (primitives)
+in 11LOGO version that we have is available at `its/doc/_info_/11logo.order document`.
+
+### MIT PLOGO
+
+In accordance with Leigh L Klotz Jr. talk at comp.lang.logo channel 
+"around 1977-1978, Gary Drescher and someone else whose name escapes me at the moment
+wrote a version of Logo in Pascal, as part of a project with Texas Instruments, for
+the TI 99/4 Home Computer, because Pascal was thefonly high-level language supported.
+They finished it and compiled it, and it produced something like a 300Kb program."  
+It was written before the TI 9900 development machine arrived and when compiled it turned
+into about 400Kb, way above the size of the 99/4, so Ed Hardebeck hand-compiled it into
+9900 assembler. 
+
+The MIT PLOGO was developed and cross-compilled on PDP-10. Lars Brinkhoff have obtained
+blessing from Gary Drescher to publish PLOGO source code.
+
+### MIT APLOGO
+
+In accordance with Leigh L Klotz Jr., Hal Abelson directed the Logo for the Apple II
+project at MIT.
+
+MIT APLOGO was developed by Stephen Hain, Patrick G. Sobalvarro and Leigh L Klotz Jr.
+It was developed and cross-compilled for the Apple-II-Plus Personal Microcomputer on
+PDP-10 at the MIT LOGO Group. It is direct predecessor for Terrapin Logo.
+We have source code for assembling Preliminary Version from 8/06/80 at `its/src/aplogo`
```

Comment:
APLOGO has been updated to an improved 1981 version.

Reply:
done.